exonum/encoding/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "exonum-encoding"
+version = "0.5.0"
+authors = ["The Exonum Team <exonum@bitfury.com>"]
+homepage = "https://exonum.com/"
+repository = "https://github.com/exonum/exonum"
+documentation = "https://docs.rs/exonum-encoding"
+readme = "README.md"
+license = "Apache-2.0"
+keywords = ["encoding", "serialization", "deserialization", "exonum"]
+categories = ["encoding"]
+description = "Serialization used in the Exonum project."
+
+[badges]
+travis-ci = { repository = "exonum/exonum" }
+
+[dependencies]
+log = "0.4.1"
+byteorder = "1.1.0"
+hex = "0.3.0"
+bit-vec = "0.4.4"
+vec_map = "0.8.0"
+rand = "0.4.2"
+serde = "1.0.10"
+serde_derive = "1.0.10"
+serde_json = "1.0.2"
+toml = "0.4.2"
+clap = "2.25.0"
+term = "0.4.6"
+colored = "1.5.2"
+env_logger = "0.5.3"
+hyper = "0.10.13"
+cookie = "0.10.1"
+router = "0.6.0"
+iron = "0.6.0"
```

Comment:
We probably don't need most of these here :)

Reply:
Yeap, it is "in progress" for a reason. 😉